neuralmonkey/decoders/transformer.py

```diff
@@ -207,27 +225,30 @@ def self_attention_sublayer(
     def encoder_attention_sublayer(self, queries: tf.Tensor) -> tf.Tensor:
         """Create the encoder-decoder attention sublayer."""
 
-        # Layer normalization
-        normalized_queries = layer_norm(queries)
-
-        # Attend to the encoder
-        # TODO handle attention histories
-        encoder_context, _ = attention(
-            queries=normalized_queries,
-            keys=self.encoder_states,
-            values=self.encoder_states,
-            keys_mask=self.encoder_mask,
-            num_heads=self.n_heads_enc,
-            dropout_callback=lambda x: dropout(
-                x, self.attention_dropout_keep_prob, self.train_mode),
-            use_bias=self.use_att_transform_bias)
-
-        # Apply dropout
-        encoder_context = dropout(
-            encoder_context, self.dropout_keep_prob, self.train_mode)
+        for i, _ in enumerate(self.encoders_states):
```

Comment:
tohle je jen jedna ze strategií, jak kombinovat enkodéry (paralelní), a podle toho, co nám vyšlo v článku, je to ta nejvíc na nic. :-)